src/pages/trails/ballots/list/BallotsList.vue

```diff
@@ -82,7 +82,12 @@ export default {
         status === true
       ) {
         this.$refs.infiniteScroll.resume();
-        this.limit += 45;
+        // Start allways with a limit of 200 and then go +100 on next query
+        if (status === true) {
```

Comment:
To me is not clear when the `status` would not be true since we're calling `this.onLoad(true)` everywhere on this page 🤔  Would you mind helping me out understanding?

Reply:
I renamed this, but this logic for filters, when we change filter we must reset out limit and positions for load